Microsoft.Toolkit.Uwp.UI.Animations/Options/RepeatOption.cs

```diff
@@ -0,0 +1,114 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Diagnostics.Contracts;
+using System.Linq;
+using Microsoft.Toolkit.Diagnostics;
+using Windows.Foundation.Metadata;
+using Windows.UI.Composition;
+using Windows.UI.Xaml.Media.Animation;
+
+#pragma warning disable CS0419
+
+namespace Microsoft.Toolkit.Uwp.UI.Animations
+{
+    /// <summary>
+    /// A type describing the repeat behavior for a custom animation.
+    /// </summary>
+    [CreateFromString(MethodName = "Microsoft.Toolkit.Uwp.UI.Animations.RepeatOption.Parse")]
+    public readonly struct RepeatOption
+    {
+        /// <summary>
+        /// The number of iterations for the animation.
+        /// </summary>
+        private readonly int value;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="RepeatOption"/> struct.
+        /// </summary>
+        /// <param name="value">The number of iterations for the animation.</param>
+        private RepeatOption(int value)
+        {
+            this.value = value;
+        }
+
+        /// <summary>
+        /// Gets a <see cref="RepeatOption"/> value representing a single iteration.
+        /// </summary>
+        public static RepeatOption One => new(1);
```

Comment:
Should this be `Once`? (I know there's other APIs that use Once as well like this one: https://docs.microsoft.com/uwp/api/windows.ui.xaml.input.keyboardnavigationmode)

Reply:
Right, good idea! 😄
Renamed in https://github.com/windows-toolkit/WindowsCommunityToolkit/pull/3639/commits/861569961203859e7c9684deeab26f5e1f05004c.